packages/manager/apps/dedicated/client/app/dedicated/server/details/translations/Messages_fr_FR.json

```diff
@@ -726,6 +726,7 @@
   "server_configuration_installation_ovh_step2_error_partition_mountPoint_wrong2": "Le point de montage /var/log n'est pas autorisé pour cette distribution. Vous devez utiliser /home/log.",
   "server_configuration_installation_ovh_step2_error_partition_mountPoint_primary": "Le point de montage c: doit être de type primaire.",
   "server_configuration_installation_ovh_step2_error_partition_mountPoint_windows": "Ce point de montage ne peut être utilisé. Il doit compris entre c et z et finir par :",
+  "server_configuration_installation_ovh_step2_error_partition_reached_max_limit": "Partition limit is reached. You can have maximum 24 partition.",
```

Comment:
It's normal that is english and not french ?

Reply:
I have asked for French content in the ticket.